### DISCUSSION TITLE
How to create a button toolbar similar to the google photos lighbox

### DISCUSSION THREAD

**Discussion by madalinul** [igordanchenko/yet-another-react-lightbox#64] [Q&A] [answered]
I am trying to create something similar to google photos where you can manipulate the image, download it and delete it but I am having a hard time adding custom elements to yarl.
<img width="1711" alt="image" src="https://user-images.githubusercontent.com/6424322/216101094-8b7ebab1-8157-435f-b96d-a753b99f0f9c.png">
I want a toolbar that has access to the current slide and it doesn't move with the slide. Is that possible?

**Comment by igordanchenko** [comment, marked as answer]
Would you mind sharing what you've tried so far that didn't work? Are there any specific issues that you ran into? 

The toolbar can be customized with the `toolbar` prop, and there are some good examples in previous discussions (e.g., [here](https://github.com/igordanchenko/yet-another-react-lightbox/discussions/40), [here](https://github.com/igordanchenko/yet-another-react-lightbox/discussions/26) and [here](https://github.com/igordanchenko/yet-another-react-lightbox/discussions/18)).
